woocommerce/woocommerce-android
# Use new FluxC code to trigger close account API request and handle errors

Part of: #8955. And continuation of [PR](https://github.com/woocommerce/woocommerce-android/pull/9033)
This PR relies on these [FluxC](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2732) changes so please don't merge until those are merged and until the base branch for this PR is `trunk`


### Description

Makes use of the new API endpoint to close account 

### Testing instructions

For now, just navigate to settings and attempt to close the account. You'll see an error because you can't close the account if you have active stores. 
**Note**: You must be logged in with a WP.com account. Application passwords login won't display this new "Close account" setting. 

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/2663464/e9f9daf6-58a3-4f1e-92c4-ed98dd84d8c8
